transit-vpc-primary-account-existing-vpc.template

```diff
@@ -80,6 +80,16 @@
       "Type" : "String",
       "Default" : ""
     },
+    "CodeLocationS3Bucket" : {
+      "Description" : "The location of the CISCO code used by this solution",
+      "Type" : "String",
+      "Default" : "solutions-reference",
+    ,
+    "SolutionsHelperS3BucketPrefix" : {
```

Comment:
The current implementation has the Solutions Helper S3 bucket appended with the region for the solutions helper lambda code.  The rest of the code is in a "solutions-reference" S3 bucket which is not region specific.  I am not 100% sure why they have this separation, but did not want to break their current functionality.